grid2op/tests/test_ma.py

```diff
@@ -1005,6 +1031,26 @@ def test_action_spaces(self):
             # check name of classes are correct
             assert re.sub("^SubGridAction", "", type(do_nothing).__name__) == re.sub("^SubGridActionSpace", "", type(ma_env.action_spaces[agent]).__name__)
 
+    
+    def test_step(self):
+        
+        np.random.seed(0)
+        # TODO seed
+        self.ma_env.reset()
+        for _ in range(10):
+            while True:
+                actions = dict(
+                    zip(
+                        self.ma_env.agents, 
+                        [self.ma_env.action_spaces[agent].sample() for agent in self.ma_env.agents]
```

Comment:
you used action_space.sample without having seeded the ma_env

Reply:
Just added the seed